### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by akorunska**
ONXP-1589 [BE] Change highlight of buttons in the list of accounts
ONXP-1590 [BE] Balance of GAS and ONYX coins is empty for all addresses
ONXP-1592 [BE] All urls must be fixed according to transition from ontology to onyx
ONXP-1597 [BE] Filtering of transaction by types in the Address page displays zero records
ONXP-1588 [BE] Remove empty Contracts from list of contracts
ONXP-1632 [BE] Add logo for Block Explorer

### CODE REVIEW

File: src/components/addresses/Detail.vue [hunk 1]
```diff
@@ -322,8 +275,9 @@
         params.contractType = 'oep8'
         this.$store.dispatch('GetAddressOep8Detail', params).then()
         if(params.assetName == "ALL"){
+          
           this.$store.dispatch('GetAddressTX',params).then()
-        }else{
+        } else{
```
Comment (arsonaut) [inline src/components/addresses/Detail.vue:280]: Just noise changes

File: src/router/index.js [hunk 2]
```diff
@@ -219,11 +180,6 @@ let routes = [
     name: 'ContractDetailwithoutpage',
     component: ContractDetail
   },
-/*   {
-    path: '/contract/:contractType/:contractHash/:pageSize',
-    name: 'ContractDetailwithoutpageSize',
-    component: ContractDetail
-  }, */
```
Comment (arsonaut) [inline src/router/index.js]: Was it already commented? It should work. Similar API is available.

File: src/router/index.js [hunk 3]
```diff
@@ -239,16 +195,6 @@ let routes = [
...
-/*   {
-    path: '/contract/RegistrationForm',
-    name: 'ContractRegistrationForm',
-    component: ContractRegistrationForm
-  },
-  {
-    path: '/contract/RegistrationForm/:net',
-    name: 'ContractRegistrationFormTest',
-    component: ContractRegistrationForm
-  }, */
```
Comment (arsonaut) [inline src/router/index.js]: Maybe leave commented?

**Review by arsonaut — APPROVED** [review]

**Review by arsonaut — COMMENTED** [review]
This change is no longer necessary
https://scaddr.apriorit.com/jira/browse/ONXP-1588?focusedCommentId=27643&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-27643

File: src/components/contracts/Detail.vue [hunk 4]
```diff
@@ -11,7 +11,7 @@
               <div v-else class="sc-no-logo-detail">C</div>
             </div>
             <div class="sc-detail-desc">
-              <h4>{{ contract.list.name }}</h4>
+              <h4>{{ contract.list.name.length !== 0 ? contract.list.name : contract.list.contract_hash }}</h4>
```
Comment (arsonaut) [inline src/components/contracts/Detail.vue]: This change is no longer necessary
https://scaddr.apriorit.com/jira/browse/ONXP-1588?focusedCommentId=27643&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-27643

File: src/components/contracts/List.vue [hunk 5]
```diff
@@ -34,7 +34,7 @@
               </td>
               <td class="font-size14 font-Regular normal_color sc-pointer"
                   @click="goToContractDetail(contract)">
-                <div class="font-blod font-size16">{{ contract.name }}</div>
+                <div class="font-blod font-size16">{{ contract.name.length !== 0? contract.name : contract.contract_hash }}</div>
```
Comment (arsonaut) [inline src/components/contracts/List.vue]: This change is no longer necessary

**Review by arsonaut — APPROVED** [review]

File: src/router/index.js (hunk 2 above)
Comment (akorunska) [reply src/router/index.js]: If so, I'll leave it commented.

**Review by arsonaut — APPROVED** [review]

**Pull request merged**